js/index.js

```diff
@@ -42,7 +50,77 @@ let currentPosition = 0;
 const carrouselArrows = document.querySelectorAll(".carrousel__arrow");
 /* Take actual image from document */
 const carrouselImage = document.querySelector(".carrousel__img");
+
+ function automatic() {
+  if (currentPosition < image.length - 1) {
+    carrouselImage.src = image[currentPosition + 1];
+    currentPosition++;
+  } else {
+    carrouselImage.src = image[0];
+    currentPosition = 0;
+  }
+  ovalBlack
+  ovalBlack();
+};
+/**
+* when the active page loads the interval function and sends the data and the activation
+* to the automatic function
+ */
+var seg;
+window.onload = function interval() {
+   seg = setInterval(automatic,3000);
+   divCont[0].classList.add('oval-black');
+}
+
+
+/**
+ *changes the color of the div's to black or gray
+*/
+ function ovalBlack(){
```

Comment:
this function could have a more indicative name like `activeOval` or `setActive`